src/canmatrix/cancluster.py

```diff
@@ -1,74 +1,84 @@
                     for receiver in frame.receivers:
-                        frameArray[index].add_receiver(receiver)
-        self._frames = frameArray
-        return frameArray
+                        frames[index].add_receiver(receiver)
+        self._frames = frames
+        return frames
 
-    def updateSignals(self):
-        signalArray = []
-        signalArrayName = []
+    def update_signals(self):  # type: () -> typing.MutableSequence[canmatrix.Signal]
+        signals = []  # type: typing.MutableSequence[canmatrix.Signal]
+        signal_names = []  # type: typing.MutableSequence[str]
         for matrixName in self:
             for frame in self[matrixName].frames:
-                for signal in frame.signals:
-                    if signal.name not in signalArrayName:
-                        signalArrayName.append(signal.name)
-                        signalArray.append(signal)
+                for signal in frame.signals:  # type: canmatrix.Signal
+                    if signal.name not in signal_names:
+                        signal_names.append(signal.name)
+                        signals.append(signal)
                     else:
-                        index = signalArrayName.index(signal.name)
+                        index = signal_names.index(signal.name)
                         for receiver in signal.receivers:
-                            signalArray[index].add_receiver(receiver)
-        self._signals = signalArray
+                            signals[index].add_receiver(receiver)
+        self._signals = signals
+        return signals
 
-    def updateECUs(self):
-        ECUArray = []
-        ECUArrayName = []
+    def update_ecus(self):  # type: () -> typing.MutableSequence[canmatrix.Ecu]
+        ecus = []  # type: typing.MutableSequence[canmatrix.Ecu]
+        ecu_names = []  # type: typing.MutableSequence[str]
         for matrixName in self:
-            for ecu in self[matrixName].ecus:
-                if ecu.name not in ECUArrayName:
-                    ECUArrayName.append(ecu.name)
-                    ECUArray.append(ecu)
-        self._ecus = ECUArray
+            for ecu in self[matrixName].ecus:  # type: canmatrix.Ecu
+                if ecu.name not in ecu_names:
+                    ecu_names.append(ecu.name)
+                    ecus.append(ecu)
+        self._ecus = ecus
+        return ecus
 
     def update(self):
-        self.updateFrames()
-        self.updateSignals()
-        self.updateECUs()
+        self.update_frames()
+        self.update_signals()
+        self.update_ecus()
 
     @property
-    def ecus(self):
+    def ecus(self):  # type: () -> typing.MutableSequence[canmatrix.Ecu]
         if not self._ecus:
-            self.updateECUs()
+            self.update_ecus()
         return self._ecus
 
     @property
-    def boardUnits(self):
+    def board_units(self):  # type: () -> typing.MutableSequence[canmatrix.Ecu]
```

Comment:
... while refactoring - pleas rename in `Ecus`

Reply:
sure, thanks for hint!